metrics/web/lib/auth/presentation/widgets/strategy/google_sign_in_option_strategy.dart

```diff
@@ -14,4 +17,16 @@ class GoogleSignInOptionStrategy implements SignInOptionStrategy {
   void signIn(AuthNotifier notifier) {
     notifier.signInWithGoogle();
   }
+
+  @override
+  MetricsButtonStyle getWidgetAppearance(
```

Comment:
Maybe we can implement this method within `SignInOptionAppearanceStrategy` abstract class since it seems like all login options should have the same design but will be able to override it if required? Anyway, currently, we have only one login option and it is not a mistake but is an interesting point to discuss I suppose. 

P.S. I suspect that this point also allows saving the previous name for the strategy - `SignInOptionStrategy`.